model/src/w3adatmd.F90

```diff
@@ -340,6 +340,9 @@ MODULE W3ADATMD
 !  7. Source code :
 !
 !/ ------------------------------------------------------------------- /
+      ! module default
```

Comment:
Most changes are white space or style and to me seem unrelated to the PR.   I'm wondering if it's worth keeping these changes in the PR or if it would be better to revert this file to its original state which would help keep this branch and develop in sync.

Reply:
implicit none should always be at the module level. Reverting this to the previous would mean reverting to a poor coding practice.

Reply:
From a code management perspective, it's preferred to keep the divergence from the develop branch to a minimum and keep the PR focused as possible.